src/AccessibilityInsights/MainWindowHelpers/TestMode.cs

```diff
@@ -196,11 +196,8 @@ private void StartLoadingSnapshot(string path, int? selectedElementId = null)
 
             var v = SelectAction.GetDefaultInstance().SelectLoadedData(path, selectedElementId);
 
-            if (selectedElementId.HasValue)
-            {
-                Logger.PublishTelemetryEvent(TelemetryAction.Hierarchy_Load_Protocol, TelemetryProperty.ProtocolMode, v.Item2.Mode.ToString());
-            }
-            Logger.PublishTelemetryEvent(TelemetryAction.Hierarchy_Load_NewFormat, TelemetryProperty.FileMode, v.Item2.Mode.ToString());
+            TelemetryProperty mode = selectedElementId.HasValue ? TelemetryProperty.ProtocolMode : TelemetryProperty.FileMode;
+            Logger.PublishTelemetryEvent(TelemetryAction.Hierarchy_Load_NewFormat, mode, v.Item2.Mode.ToString());
```

Comment:
This is probably intentional, but just to confirm - the original code sends 2 events if `selectedElementId.HasValue` is true, whereas this new code only sends 1 event. Is this as expected?

Reply:
Yes. They can be distinguished by the underlying property, if needed